shared/macros/10-bash.jinja

```diff
@@ -1586,6 +1586,10 @@ if ! grep "$mount_point_match_regexp" /etc/fstab; then
     previous_mount_opts=$(grep "$mount_point_match_regexp" /etc/mtab | head -1 |  awk '{print $4}' \
                 | sed -E "s/(rw|defaults|seclabel|{{{ mount_opt }}})(,|$)//g;s/,$//")
     [ "$previous_mount_opts" ] && previous_mount_opts+=","
+    # In iso9660 filesystems mtab could describe a "blocksize" value, this should be reflected in fstab as "block"
+    if [  "{{{ type }}}" == "iso9660" ] ; then
```

Comment:
```suggestion
    # In iso9660 filesystems mtab could describe a "blocksize" value, this should be reflected in
    # fstab as "block".  The next variable is to satisfy shellcheck SC2050.
    fs_type="{{{ type }}}"
    if [  "$fs_type" == "iso9660" ] ; then
```
This is one option to make the shellcheck SC2050 warning disappear. The comment line was broken to keep it within 99 columns.